youtube_dl/extractor/promptfile.py

```diff
@@ -33,7 +33,12 @@ def _real_extract(self, url):
             raise ExtractorError('Video %s does not exist' % video_id,
                                  expected=True)
 
+        chash_pattern = r'\$\("#chash"\)\.val\("(.+)"\+\$\("#chash"\)'
+        chash = self._search_regex(chash_pattern, webpage, "chash")
         fields = self._hidden_inputs(webpage)
+        k = fields.keys()[0]
```

Comment:
`keys()` is not a list under python 3. There is also no guarantee `chash` is the only in `fields` dict.